dspace/config/modules/authentication-shibboleth.cfg

```diff
@@ -38,7 +38,7 @@
 authentication-shibboleth.lazysession = true
 
 # The url to start a shibboleth session (only for lazy sessions)
-authentication-shibboleth.lazysession.loginurl = /Shibboleth.sso/Login
+authentication-shibboleth.lazysession.loginurl = ${dspace.baseUrl}/Shibboleth.sso/Login
```

Comment:
If it's not specified, it would be a local-only logout.
This will leave at least system administrators with the choice on how to setup their DSpace (based on policies)
This would be a redirect